src/styles/blocks/product__card.scss

```diff
@@ -0,0 +1,124 @@
+.product__card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid rgb(243, 243, 243);
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  transition: all 0.3s ease-in-out;
```

Comment:
It's a good practice to avoid using the 'all' keyword in transitions for performance reasons. Specify the properties you want to transition, such as 'transform'.